stacks_queues/stack.py

```diff
@@ -21,18 +21,29 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store == None:
+            raise StackEmptyException("The stack is already empty!")
+
+        elem = self.store.get_first()
+        self.store.remove_first()
+
+        return elem
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        if self.store.get_first() == None:
+            return True
+
+        return False
+
 
     def __str__(self):
```

Comment:
✨ This works, however the Linked List class, has a `__str__` method you can utilize!